src/main/scala/org/broadinstitute/dsde/agora/server/dataaccess/mongo/AgoraMongoDao.scala

```diff
@@ -83,6 +76,33 @@ class AgoraMongoDao(collection: MongoCollection) extends AgoraDao {
     }
   }
 
+  override def find(entity: AgoraEntity, projection: Option[AgoraEntityProjection]): Seq[AgoraEntity] = {
+    projection match {
+      case Some(project) => find(EntityToMongoDbObject(entity), projection)
```

Comment:
mostly did this to avoid shadowing.. but I agree a different name seems more appropriate